srcDoc/release_notes.rst

```diff
@@ -70,6 +70,14 @@ and PDF documentation from the rst files in srcDoc.
 What's Fixed in Master
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Fix incorrect behavior of -unbounded with small data (#125)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In previous releases, if -unbounded was provided with small data, the execution
+fail due to a swich off of unbounded after setting large data types.
+
+Now, unbounded will be not turn off when small data is used.
```

Comment:
```
In the 2.4 release a regression was introduced and the use of `-unbounded` would cause a failure when using datasizes from `28` to `63000 Bytes`. This issue has been resolved.
```

Reply:
Done